dts/bindings/usb/usb-audio-hs.yaml

```diff
@@ -128,7 +128,7 @@ properties:
     required: false
     description: Enable Bass Boost feature.
                  Currently not supported.
-  mic-mic-feature-loduness:
+  mic-feature-loduness:
```

Comment:
This should be `-loudness`.

Reply:
Even more typos...

Reply:
Indeed:
https://github.com/zephyrproject-rtos/zephyr/blob/d1bdc7707b32c76bed23343a392a8c2bb0df69f0/dts/bindings/usb/usb-audio-hp.yaml#L110
https://github.com/zephyrproject-rtos/zephyr/blob/d1bdc7707b32c76bed23343a392a8c2bb0df69f0/dts/bindings/usb/usb-audio-mic.yaml#L122